tasks/farm5/Alignment.wdl

```diff
@@ -0,0 +1,86 @@
+version 1.0
+
+## Copyright Wellcome Sanger Institute, Oxford University, and the Broad Institute 2020
+##
+## This WDL pipeline implements the Short Read Alignment Pipeline as described in
+## https://github.com/malariagen/pipelines/blob/c7210d93628aaa31f26baa88a92e10322368b78e/docs/specs/short-read-alignment-vector.md
+## This is an initial proof of concept implementation.  It is designed to ONLY work on one sample, with ONLY its
```

Comment:
Looks like this comment will need an expansion to mention the other input types it can use

Reply:
Yeah, that needs an update